flink-runtime-web/web-dashboard/src/app/interfaces/job-checkpoint.ts

```diff
@@ -113,7 +111,7 @@ export interface CheckPointTaskStatistics {
 }
 
 export interface CheckPointConfig {
-  mode: SafeAny;
+  mode: 'exactly_once' | string;
```

Comment:
Why the special role for `exactly_once`?

Reply:
That's the only value I could find in the code, but I assume there's more, so I documented the known value and played it safe (IDEs can still auto-complete that value).

Reply:
There seems to be only this and `at_least_once` in Flink, but yeah, not for this PR to resolve...